test/AElf.Runtime.CSharp.Tests/ContractAuditorTests.cs

```diff
@@ -157,6 +166,13 @@ public void CheckBadContract_ForDeniedMemberUseInSeparateClass()
                 .ShouldNotBeNull();
         }
         
+        [Fact]
+        public void CheckBadContract_ForLargeArrayInitialization()
+        {
+            _findings.FirstOrDefault(f => f.GetType() == typeof(ArrayValidationResult) && f.Info.ReferencingMethod == "InitLargeArray")
```

Comment:
May be we can use `f is ArrayValidationResult`

Reply:
Fixed by the latest commit